src/test/e2e/Tests/e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { Browser, Page } from 'puppeteer';
+
+const puppeteer = require('puppeteer');
+const HomePage = require('../PageObjects/HomePage');
+const homePage = new HomePage;
+let browser: Browser;
+let page: Page;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch({
+    headless: false, // set to false and uncomment slowMo and args to see tests in realtime
```

Comment:
this is still a lot of config in the test suite, this needs moving to a puppeteer config file somewhere